picli/configs/base_pipe.py

```diff
@@ -55,38 +56,27 @@ def _read_group_vars(self):
         the other group_vars files to override the values in all.yml
         :return: list
         """
+        group_vars_dir = f'{self.base_config.vars_dir}/group_vars.d'
 
         group_configs = []
-        if os.path.isdir(f'{self.base_config.vars_dir}/group_vars.d/'):
+        if os.path.isdir(group_vars_dir):
             for root, dirs, files in os.walk(
                     f'{self.base_config.vars_dir}/group_vars.d/'
             ):
                 if len(files) == 0:
-                    message = f'group_vars is blank in {self.base_config.vars_dir}'
+                    message = f'No group_vars found in {self.base_config.vars_dir}'
                     util.sysexit_with_message(message)
                 for file in files:
-                    if file == 'all.yml':
-                        with open(os.path.join(root, file)) as f:
-                            group_config = f.read()
-                            group_configs.append(util.safe_load(group_config))
+                    with open(os.path.join(root, file)) as f:
+                        group_config = f.read()
+                        group_configs.append(
+                            {'file': file, 'config': util.safe_load(group_config)}
```

Comment:
consider letting yaml.safe_load read the file itself here instead of passing in a string... less code